vmdb/app/models/ems_microsoft.rb

```diff
@@ -96,7 +96,8 @@ def execute_power_operation(cmdlet, vm_uid_ems, *parameters)
     return unless vm_uid_ems.guid?
 
     params  = parameters.join(" ")
-    command = "powershell #{cmdlet}-SCVirtualMachine -VM (Get-SCVirtualMachine -ID #{vm_uid_ems}) #{params}"
+    command = "powershell Import-Module VirtualMachineManager; Get-VMMServer #{ipaddress}; "
```

Comment:
@jrafanie
I think this has become a general question.  Given that Nick's code change will be run on the SCVMM server "localhost" will be used, like all other cmdlets we run, instead of an IP.